sdcm/chaos_mesh.py

```diff
@@ -0,0 +1,62 @@
+import json
+import logging
+
+from sdcm.cluster_k8s import KubernetesCluster
+from sdcm.utils.k8s import HelmValues, KubernetesOps
+
+LOGGER = logging.getLogger(__name__)
+
+
+class ChaosMesh:  # pylint: disable=too-few-public-methods
+    NAMESPACE = "chaos-mesh"
+    VERSION = "2.5.0"
+    HELM_SETTINGS = {'dashboard.create': True, 'dnsServer.create': True}
```

Comment:
dashboard I understand

what's the dns server for ?

Reply:
it's enablement for dns server faults - I can disable it for now as it's not needed yet.

Reply:
Please, define the config hierarchically. So, it will be easier to update it later with minimal code line changes.

Reply:
what do you mean by 'hierarchically'?

Reply:
> what do you mean by 'hierarchically'?

```
HELM_SETTINGS = {
    'dashboard.create': True,
    'dnsServer.create': True,
}
```